packages/aws-cdk/test/integ/cli/cloud-assemblies/0.36.0/cdk.out

```diff
@@ -0,0 +1 @@
+{"version":"0.36.0"}
```

Comment:
Yes it is. I think this file exists for backwards compatibility with some old version. Elad will know.

Reply:
I think that is correct.

Reply:
Indeed. It is there since old CLIs would look for this file name in the output directory. The `version` field here will tell them that they can't do anything besides yelling. I assume we can probably get rid of it by now...